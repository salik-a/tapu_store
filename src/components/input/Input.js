import React from "react";
import { InputContainer, Input } from './InputStyled'

const TextInput = ({ placeholder, onChangeText, value, isSecure }) => {
    return (
        <InputContainer >
            <Input
                placeholder={placeholder}
                onChangeText={onChangeText}
                value={value}
                secureTextEntry={isSecure}
            />
        </InputContainer>
    );
};

export default TextInput;