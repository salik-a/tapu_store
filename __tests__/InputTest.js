import React from 'react'
import { fireEvent, render, waitFor } from '@testing-library/react-native'
import Input from '../src/components/input/Input';
test('should match with snapshot', () => {

    const comp = render(<Input />);
    expect(comp).toMatchSnapshot();
});

