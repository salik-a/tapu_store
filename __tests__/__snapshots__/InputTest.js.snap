// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "center",
        "borderBottomWidth": 0.8,
        "borderColor": "#BBC3CF",
        "height": 50,
        "marginVertical": 20,
        "width": "85%",
      },
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
        },
      ]
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
